tests/unit/test_pool.py

```diff
@@ -342,6 +447,24 @@ def test_get_non_empty_session_exists(self):
         self.assertTrue(session._exists_checked)
         self.assertTrue(pool._sessions.empty())
 
+    def test_spans_get_non_empty_session_exists(self):
+        pool = self._make_one()
+        database = _Database("name")
+        previous = _Session(database)
+        pool.bind(database)
+        with trace_call("pool.Get", previous):
+            pool.put(previous)
+            session = pool.get()
+            self.assertIs(session, previous)
+            session.create.assert_not_called()
+            self.assertTrue(session._exists_checked)
+            self.assertTrue(pool._sessions.empty())
+
+        self.assertSpanAttributes(
```

Comment:
can you also add assertion on span events here?

Reply:
Done, thanks!